01/highest-rated.md

```diff
@@ -178,6 +186,18 @@ Write the logical steps here.
 
 ##### !end-placeholder
 
+### !explanation
+
+Example Steps:
+
+1. Check if restaurants is empty
+    - if it's empty, return `None`
+2. Set `highest_rated` to the first element in `restaurants`
+3. Loop through restaurants and compare the current resturant rating to the `highest_restaurant` rating.
+    - If the current restaurant rating is higher, assign it to `highest_restaurant`
+4. Return `highest_rated`
+
+### !end-explanation
```

Comment:
I purposefully increased the width of the table because the explanations were long. Do you think it looks too weird?

Reply:
It does strike me as looking off since there's barely any margin and it's so different than all the other lessons. I do see what you're saying about the explanation column having way more content than the other columns. Is there a way we can reformat this to include a short explanation and more detailed explanation below? 

It might also be sufficient to just add slightly more padding between the sides of the table and the edges of the screen.